WcaOnRails/db/migrate/20171122010954_remove_rubiks.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class RemoveRubiks < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Two things:

1. We need to update events.seeds.rb as well.
2. I personally think it gets annoying to make sure that events.seeds.rb is in sync with this migration, so it's easier for this migration to just empty the events table entirely and reload from seeds. See https://github.com/jfly/worldcubeassociation.org/blob/relaxing-proximity-policy/WcaOnRails/db/migrate/20170831170616_fix_multiple_countries.rb for an example of this.